transformations/slangificator/transformation.py

```diff
@@ -217,6 +172,8 @@ def __init__(self, seed=0, max_outputs=1):
 
         pathDic = os.path.dirname(os.path.abspath(__file__))
 
+        self.nlp = spacy.load("en_core_web_sm")  # get an instnace of the tokenizer
+
```

Comment:
You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).

Reply:
Got it, just modified the code that way